### PR TITLE
Adding Epion integration documentation

### PR DESCRIPTION
**Pull request by lhgravendeel**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/107570
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5055
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/epion.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Requires a configured Epion Air device and an Epion account with access to this device.
+
+## Prerequisites
+
+Epion API token setup.
+
+1. Login to [Epion](https://www.epion.nl/).
+2. Select [Integrations](https://epion.nl/dashboard/integrations) in the left menu.
+3. Click on "Generate new Token" to create a new token, if you don't see one already.
```
Comment (c0ffeeca7) [inline source/_integrations/epion.markdown]:
```suggestion
3. To create a new token, if you don't see one already, select **Generate new Token**.
```

File: source/_integrations/epion.markdown [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+4. Copy your generated token. This will typically be an alphanumerical values with hyphens that starts with "a/"
```
Comment (c0ffeeca7) [inline source/_integrations/epion.markdown]:
```suggestion
4. Copy your generated token. The token is a set of alphanumeric characters separated by hyphens. It starts with "a/".
```

File: source/_integrations/epion.markdown [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+The Epion integration can now be activated using this API token.
+
+All connected devices will show entities for the Carbon Dioxide (CO2) concentration, temperature, relative humidity and barometric pressure.
```
Comment (c0ffeeca7) [inline source/_integrations/epion.markdown]:
```suggestion
All connected devices will show entities for the carbon dioxide (CO2) concentration, temperature, relative humidity, and barometric pressure.
```

**Comment by c0ffeeca7** [comment]
Tiny tweaks

**Comment by frenck** [comment]
I've marked this PR, as changes are requested that need to be processed.
Please un-draft it once it is ready for review again by clicking the **"Ready for review"** button.

Thanks! 👍 

../Frenck

[_Learn more about our pull request process._](https://developers.home-assistant.io/docs/review-process#prs-are-being-drafted-when-changes-are-needed)

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @lhgravendeel :+1:

**Pull request merged**